README.md

```diff
@@ -7,6 +7,24 @@ Confluent API.
 
 ## Getting Started
 
+### Fix for make generate
+The Confluent Provider Terraform documentation has a missing block that will cause the `make generate` command to fail with the following error:
+```shell
+scraper: error: Failed to scrape Terraform provider metadata: cannot scrape Terraform registry: failed to scrape resource metadata from path: ../.work/confluentinc/confluent/docs/resources/confluent_ksql_cluster.md: failed to find the prelude of the document using the xpath expressions: //text()[contains(., "description") and contains(., "page_title")]
+```
+
+To fix this, run:
+```shell
+echo '---                                                                                                                      (vcluster_i534922_i534922_shoot--pi--vcluster-host/default)
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "confluent_ksql_cluster Resource - terraform-provider-confluent"
+subcategory: ""
+description: |-
```

Comment:
Why is your vcluster context specified here?

Reply:
Copy & paste mistake from my terminal showing the context on the far right 😃